### PR TITLE
Frontend files layout, adding register profile page

### PR DESCRIPTION
**Pull request by 6rivu**
Designed the basic register profile page with css.

### CODE REVIEW

File: src/pages/register.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+const Register = (props) => {
+
+  //State variables
+  const [walletAddress, setWallet] = useState("");
+  const [status, setStatus] = useState("");
+  const [description, setDescription] = useState("");
+  const [name, setName] = useState("");
+  const [url, setURL] = useState("");
+
+  const connectWalletPressed = async () => { //TODO: connect to wallet
```
Comment (gabsong) [inline src/pages/register.js]: nit: I think you can just name this connectWallet

File: src/pages/register.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+   
+  };
+
+  const onMintPressed = async () => { //TODO: mint profile token
```
Comment (gabsong) [inline src/pages/register.js]: Also just call this mint

File: src/pages/register.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    
+  };
+
+  return (
+    <div className="Minter">
+      <button id="walletButton" onClick={connectWalletPressed}>
+        {walletAddress.length > 0 ? (
+          "Connected: " +
+          String(walletAddress).substring(0, 6) +
```
Comment (gabsong) [inline src/pages/register.js]: Is the casting to String really needed? Based on the previous use of `walletAddress`, it seems to be a string already.

**Review by gabsong — APPROVED** [review]
Approved with small comments.

**Comment by gabsong** [comment]
One more thing: This PR seems to be a fork, then merging from the fork. Actually, the workflow we're using is that you can create a branch on @ewerx 's repo. So next time, instead of forking from your repo, close Ehsan's repo, you can create a branch, and do a PR from the branch!

File: src/pages/register.js (hunk 3 above)
Comment (6rivu) [reply src/pages/register.js]: Not sure about it. I think it'll depend on what data we extract from web3.js/ether.js object. Removing the casting for now.

**Pull request merged**